CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+# Contribuindo
+
+Você pode contribuir com o projeto de diversas formas, implementando uma
+funcionalidade nova, corrigindo um bug, procurando bugs, revisando pull
+requests, entre outras.
+Para se inteirar do projeto, entre no
+[Discord](https://discord.gg/vjZS6BQXvM) e participe das discussões.
+
+## 🤝 Contribuindo com atividades que não são de código
+
+O projeto precisa de ajuda em diversas frentes diferentes, como QA, produto,
+design e gestão. Entre no servidor do Discord, onde há canais específicos para
+essas atividades.
+
+Se você encontrou um bug, vá nas
+[issues](https://github.com/SOS-RS/backend/issues)
+do projeto e reporte-o lá. Verifique antes se já não existe um bug aberto para o
+problema que quer relatar, usando a busca. O mesmo vale para novas
+funcionalidades.
+
+O restante deste documento focará nas contribuições de código.
+
+## ✅ Escolhendo qual será sua contribuição de código
+
+Verifique no [projeto do Github](https://github.com/orgs/SOS-RS/projects/1)
+quais funcionalidades ainda não foram implementadas e já estão prontas para
+serem desenvolvidas, elas estarão na coluna "Disponível pra dev". Lá há itens de
+backend e frontend, então atente-se para qual você gostaria de participar.
```

Comment:
```suggestion
serem desenvolvidas, elas estarão na coluna "Disponível pra dev". Nesse projeto existem itens de
`backend` e `frontend`, então atente-se para qual você gostaria de participar.
```

Reply:
Não acho que precisamos marcar backend e frontend no formato de código. Poderia ser itálico, por ser lingua estrangeira, mas não estamos fazendo isso em outros lugares...